plugins/modules/nameserver_info.py

```diff
@@ -57,6 +57,14 @@
             - How often to retry on SERVFAIL errors.
         type: int
         default: 0
+    servers:
+        description:
+            - A list of DNS server IP addresses to use for resolving nameserver information.
+            - When specified, these servers are used instead of the system's default DNS settings.
```

Comment:
Doesn't this have to be a list of IPs?
```suggestion
            - When specified, these servers are used instead of the system's default DNS settings.
            - This must be a list of IP addresses.
```
In any case, you also should add an example for this.